### DISCUSSION TITLE
Slicing bug(?). Orca closing/bridging holes.

### DISCUSSION THREAD

**Discussion by xxxWhatzitTooya** [OrcaSlicer/OrcaSlicer#4684] [Q&A]
I don't have close holes on, enabled and disabled every settings combination I could think of to no avail.
Creality Slicer and Cura slice it like expected.
The printer is a K1 Max with a 0.6 nozzle and I tried with Orca Slicer 1.9,1 and 2.0.0-rc and different layer widths between 0.4 and 0.75.

Sliced with Orca Slicer 2.0.0-rc.

I also used a portable version in a sandbox so every setting was stock and still got this issue.
Edit: The model I use is [this one](https://www.printables.com/de/model/686750-k1-max-left-side-mounted-tool-holder-remix).

![Screenshot 2024-03-24 142522 - Copy](https://github.com/SoftFever/OrcaSlicer/assets/89413786/4c78bf01-5f22-4c6d-ad0f-83a07d4f067f)
![Screenshot 2024-03-24 142630](https://github.com/SoftFever/OrcaSlicer/assets/89413786/4712758e-1da5-4215-9fc0-72a4e5c7ca04)

**Comment by 4eIIIuP** [comment]
Try to fix the model using the right mouse button on the model - > "Fix the model"
![image](https://github.com/SoftFever/OrcaSlicer/assets/158848753/6f3e7f16-eb67-4f63-aba3-b22d8c0139f8)

**Comment by mauledbyjesus** [comment]
Just ran into this. "Make overhangs printable" was the culprit in my case.
